### PR TITLE
Update Vizio docs to reflect improved discovery

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
The discovery function built into `pyvizio` now includes port information, so updating the docs to reflect that, which will hopefully reduce the time it takes for a user to set up the integration for the first time

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/vizio.markdown [hunk 1]
```diff
@@ -81,21 +71,21 @@ To add your Vizio TV to your installation, add the following to your `configurat
...
-    host: IP_ADDRESS
-    access_token: AUTH_TOKEN
+    host: "DEVICE_IP:DEVICE_PORT"
+    access_token: YOUR_AUTH_TOKEN
 ```
 
 {% configuration %}
 host:
-  description: IP address of your device.
+  description: `IP Address:Port` for your device (port is optional but recommended).
```
Comment (frenck) [inline source/_integrations/vizio.markdown]:
```suggestion
  description: "`IP Address:Port` for your device (port is optional but recommended)."
```

File: source/_integrations/vizio.markdown [hunk 2]
```diff
@@ -81,21 +71,21 @@ To add your Vizio TV to your installation, add the following to your `configurat
...
   required: true
   type: string
 access_token:
   description: Authentication token you received in the last step of the pairing process (if applicable).
   required: false
   type: string
 device_class:
-  description: The class of your device. Valid options are `tv` or `soundbar`
+  description: The class of your device. Valid options are `tv` or `soundbar`.
```
Comment (frenck) [inline source/_integrations/vizio.markdown]:
```suggestion
  description: "The class of your device. Valid options are `tv` or `soundbar`."
```

File: source/_integrations/vizio.markdown (hunk 1 above)
Comment (raman325) [reply source/_integrations/vizio.markdown]: 🤦‍♂ thanks @frenck this is why I shouldn't submit PR's right before bedtime

**Review by frenck — APPROVED** [review]
Thanks! 👍

**Pull request merged**